### ISSUE TITLE
[issue]: Unable to start graphical interface in SystemRescueCD on laptop with legacy nvidia gpu

### ISSUE THREAD

**Issue by peter099** [ventoy/Ventoy#1536]
### Official FAQ

- [X] I have checked the official FAQ.

### Ventoy Version

1.0.72

### What about latest release

Yes. I have tried the latest release, but the bug still exist.

### BIOS Mode

Legacy BIOS Mode

### Partition Style

MBR

### Disk Capacity

8GB

### Disk Manufacturer

Kingston

### Image file download link (if applicable)

https://www.system-rescue.org/Download/

### What happened?

I have laptop HP 8560w with gpu nvidia Quadro 1000M. I am trying to boot SystemRescueCD but when i execute 'startx' it shows black screen. I think the problem is ventoy because if i create the bootable usb with Etcher i don't have problem to start the graphical interface.

**Comment by crasadure** [comment]
@peter099 
I tested systemrescue-9.01-amd64.iso file on virtual machine...ventoy is loading ISO file on legacy mode
watch my test video pls : https://vimeo.com/692840791

**Comment by peter099** [comment]
Maybe booting in virtual machine is not the same as booting on real hardware. I have tested version 9.01, 8.05 and 8.04 and getting the same problem with all of them.

**Comment by crasadure** [comment]
I tested it at school computer on real machine on Legacy mode today..it is booting ISO file successfully. I think your computer have a problem guy

**Comment by peter099** [comment]
May be is the specific hardware on my laptop HP 8560W, but i created another bootable usb with Ventoy and again i can't start the graphical environment in SystemRescueCD.

**Comment by htcfreek** [comment]
Did you tried the cli mode of ventoy? This helped me with some isos.

**Comment by peter099** [comment]
No, i haven't tried the cli mode. I doubt this will fix the issue, but i will try it.
